samples/03.speech/a.direct-line-speech/index-directline-http.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
```

Comment:
All samples must come with `README.md`.

Reply:
@rohit21agrawal Since we don't want to expose this API to other developers, can you remove the sample all together?